source/util/encode_uri_parameters.ts

```diff
@@ -1,6 +1,6 @@
 // :copyright: Copyright (c) 2018 ftrack
 
-function encodeUriParameters(data) {
```

Comment:
I think remove it and to use `URLSearchParams`

Reply:
fixed!